tap_shopify/__init__.py

```diff
@@ -24,6 +24,7 @@ def initialize_shopify_client():
     version = '2021-04'
     session = shopify.Session(shop, version, api_key)
     shopify.ShopifyResource.activate_session(session)
+    return shopify.Shop.current().attributes
```

Comment:
Add a code comment for the functionality achieved in this line

Reply:
Added helper comment as per suggestion.